ftd/built-in-types.ftd

```diff
@@ -817,6 +818,41 @@ within its container area without cropping or stretching the image.
 - `length`: This anonymous record value will set the dimensions of the
 background image based on the specified x and y values.
 
+
+
+
+
+
+--ds.h2: `ftd.fetch-priority`
+
+The `ftd.fetchPriority` property of the HTMLImageElement interface represents a hint given to the browser on
+how it should prioritize the fetch of the image relative to other images.
+
+-- ds.code: `ftd.fetch-priority`
+lang: ftd
+
+\-- or-type fetch-priority:
+
+\-- constant string auto: auto
+\-- constant string high: high
+\-- constant string low: low
+
+-- ds.markdown
+
+As shown above, the `ftd.fetch-priority` has following variants
+
+- `auto`: Default mode, which indicates no preference for the fetch priority.
+The browser decides what is best for the user.
```

Comment:
I am not fond of `auto`. We already have `optional`, if you want to pass nothing, pass NULL.